pkg/agent/agent_windows.go

```diff
@@ -0,0 +1,41 @@
+// +build windows
+
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package agent
+
+import (
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/util"
+)
+
+// setupExternalConnectivity installs OpenFlow entries to SNAT Pod traffic using Node IP, and then Pod could communicate
+// to external IP addresses.
+func (i *Initializer) setupExternalConnectivity() error {
+	brName := i.ovsBridgeClient.GetBridgeName()
+	subnetCIDR := i.nodeConfig.PodCIDR
+	nodeIP := i.nodeConfig.NodeIPAddr.IP
+	// Enable IP-Forwarding on the interface of OVS bridge.
```

Comment:
Why this is needed?
Please add some comments to explain.

Also I feel this sounds like overlapping with routeClient functionalities?

Reply:
Enable IP-Forwarding is because SNAT could modify the source IP to the Node IP, but we still need to leverage the host networking stack to fill a correct dst MAC, otherwise the packet might be forwarded by mistake after it left the host.

Reply:
Could you add some comments to code to explain why IP forwarding should be enabled?

I do not mean the whole setupExternalConnectivity(). OVS flows seems irrelevant to routeClient, but enabling IP forwarding seems overlap with what routeClient does?

Reply:
Sure, will update.